ui/lib/pki/addon/components/page/pki-tidy-status.hbs

```diff
@@ -28,9 +18,87 @@
   </ToolbarActions>
 </Toolbar>
 
+<Hds::Alert
+  @type="inline"
+  @color={{this.tidyStateAlertBanner.color}}
+  class="has-top-margin-m"
+  @icon={{this.tidyStateAlertBanner.icon}}
+  as |A|
+>
+  <A.Title>{{this.tidyStateAlertBanner.title}}</A.Title>
+  <A.Description>{{this.tidyStateAlertBanner.message}}</A.Description>
+  {{#if this.tidyStateAlertBanner.shouldShowCancelTidy}}
+    <A.Button @text="Cancel tidy" @color="critical" {{on "click" (perform this.cancelTidy)}} />
+  {{/if}}
+  {{#if @tidyStatus.responseTimestamp}}
+    <A.Description class="has-top-margin-xs">
+      Updated
+      {{date-format @tidyStatus.responseTimestamp "MMM d yyyy, h:mm:ss aaa" withTimeZone=true}}
+    </A.Description>
+  {{/if}}
+</Hds::Alert>
+
+{{#each this.generalSectionFields as |attr|}}
+  {{#if (or (eq attr "time_started") (eq attr "time_finished") (eq attr "last_auto_tidy_finished"))}}
```

Comment:
Mostly as a handlebar FYI you could replace the conditional logic here leverage the `@formatDate` arg to do something like 

```js
<InfoTableRow
  @label={{humanize (dasherize attr)}}
  @formatDate={{if (includes attr (array "time_started" "time_finished" "last_auto_tidy_finished")) "MMM dd, yyyy hh:mm:ss a"}}
  @value={{get @tidyStatus attr}}
  @alwaysRender={{true}}
    />